crates/ruff_server/src/server/api/requests/format.rs

```diff
@@ -95,6 +95,16 @@ fn format_text_document(
             tracing::debug!("Ignored path via `{}`: {}", exclusion, file_path.display());
             return Ok(None);
         }
+
+        if let Some(inclusion) = match_any_inclusion(
+            &file_path,
+            &file_resolver_settings.include,
+            &file_resolver_settings.extend_include,
+        ) {
+            tracing::debug!("Included path via `{}`: {}", inclusion, file_path.display());
+        } else {
+            return Ok(None);
+        }
```

Comment:
It seems that we now always have the same block

1. check for exclusion
2. check for inclusion

Can we extract this into a `check_file_included` which does both rather than repeating the two checks in every call site?

Reply:
Yeah, I've done so in a new `resolve.rs` file with `is_document_excluded` function.